pkg/anago/release.go

```diff
@@ -525,6 +535,20 @@ func (d *DefaultRelease) CreateAnnouncement() error {
 	return nil
 }
 
+// GoVersion gets the Go version used to build the release, which is added to
+// the announcement
+func (d *defaultReleaseImpl) GoVersion() (string, error) {
+	const semVerRegex = `^?([0-9]+)(\.[0-9]+)?(\.[0-9]+)`
+
+	cmdStatus, err := command.New("go", "version").RunSilentSuccessOutput()
```

Comment:
I think we can use `runtime.Version()` instead of the command execution.

Reply:
That's a good point. I'll update the PR.